src/modules/local_position_estimator/sensors/vision.cpp

```diff
@@ -6,8 +6,8 @@ extern orb_advert_t mavlink_log_pub;
 
 // required number of samples for sensor
 // to initialize
-static const uint32_t 		REQ_VISION_INIT_COUNT = 20;
-static const uint32_t 		VISION_TIMEOUT =    500000;	// 0.5 s
+static const uint32_t 		REQ_VISION_INIT_COUNT = 5;
+static const uint32_t 		VISION_TIMEOUT =    10000000;	// 10 s
```

Comment:
@jgoppert Switching Flow to velocity also helps me a lot e.g. in the case of indoors/outdoors transition (thanks for that), when onboard vision might not be available anymore. I am just curious, why VISION_TIMEOUT now is set to 10s - indeed I reduced that to 0.1 seconds to let LPE know as soon as possible that vision is not available anymore.